crates/dbsp/src/operator/dynamic/join.rs

```diff
@@ -1329,9 +1329,13 @@ where
         }
 
         self.stats.output_tuples += output_tuples.len();
+
         // Sort `output_tuples` by timestamp and push all tuples for each unique
         // timestamp to the appropriate batcher.
-        output_tuples.sort_by_key();
+        // Skip this step when using unit timestamps.
```

Comment:
the comment says "sort by timestamp", but the function is called "sort_by_key".

Reply:
yes, because the timestamp is the key in this array of key-value pairs